pkg/services/searchV2/service.go

```diff
@@ -41,20 +41,33 @@ type dashMeta struct {
 func (s *StandardSearchService) DoDashboardQuery(ctx context.Context, user *backend.User, orgId int64, query DashboardQuery) *backend.DataResponse {
 	rsp := &backend.DataResponse{}
 
-	if user == nil || user.Role != string(models.ROLE_ADMIN) {
-		rsp.Error = fmt.Errorf("search is only supported for admin users while in early development")
-		return rsp
-	}
-
 	// Load and parse all dashboards for given orgId
 	dash, err := loadDashboards(ctx, orgId, s.sql)
 	if err != nil {
 		rsp.Error = err
 		return rsp
 	}
 
-	// TODO!!! get user from context?
-	dash, err = s.applyAuthFilter(nil, dash)
+	getSignedInUserQuery := &models.GetSignedInUserQuery{
```

Comment:
Smart :) Maybe leave a comment to not forget to do sth with this in the future?

Reply:
yup, readded!